manifests/i/IsaqueS/AventurasDeQuintal/0.8.1/IsaqueS.AventurasDeQuintal.locale.pt-BR.yaml

```
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: IsaqueS.AventurasDeQuintal
+PackageVersion: 0.8.1
+PackageLocale: pt-BR
+Publisher: Isaque Sebold de Almeida

```

```suggestion
Publisher: isaque
```
to match Control Panel.